odoo/addons/base/models/res_partner.py

```diff
@@ -150,7 +150,7 @@ def default_get(self, default_fields):
             values['lang'] = values.get('lang') or parent.lang or self.env.lang
         return values
 
-    name = fields.Char(index=True)
+    name = fields.Char(index=True, default_export_for_import=True)
```

Comment:
``default_export_compatible`` maybe ?

Reply:
Thanks, I know it is not a top name but it means field to be exported when we need an export compatible for import. For example, we can export "display_name" in contact but this field is not compatible for importation.
As you both agreed I will change. (don't hesitate if you come up with a better name)